am-lib/build.gradle

```diff
@@ -76,18 +76,22 @@ dependencies {
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.8'
     compile group: 'org.hibernate.validator', name: 'hibernate-validator', version: '6.0.16.Final'
     compile group: 'org.aspectj', name: 'aspectjrt', version: '1.9.2'
+    compile group: 'com.google.guava', name: 'guava', version: '27.1-jre'
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc', version: '2.1.3.RELEASE'
```

Comment:
As discussed these dependencies should not be in main classpath as this way they would be included in library jar.